src/redux/store/initialized.ts

```diff
@@ -0,0 +1,19 @@
+import { AnyAction } from 'redux';
+
+const SET_INITIALIZED = 'SET_INITIALIZED';
+
+export const setInitialized = (value: boolean) => ({
+  type: SET_INITIALIZED,
+  value,
+});
+
+const initializedReducer = (initialized = false, action: AnyAction) => {
```

Comment:
I meant the name of the reducer -> `initialReducer` - maybe you meant this

Reply:
✅